jsonnet/rules.jsonnet

```diff
@@ -102,6 +102,10 @@
               severity: 'critical',
             },
           },
+          {
+            expr: 'sum((cluster:master_nodes * on(node) group_left kube_node_labels) or on(node) (kube_node_info * on(node) group_left kube_node_labels)) BY (node, kernel_version, os_image, label_node_openshift_io_os_id, label_beta_kubernetes_io_instance_type, label_node_role_kubernetes_io)',
```

Comment:
We don’t need the first part of the query, as master nodes are included in the kube_pod_info metric. The second half aggregates by the “node” label values which causes the cardinality of this recording rule to be the cardinality of number of nodes, which I don’t think you intended to do as these are meant to be aggregates. I think you want to remove the node label from the by clause.

Reply:
Can do that but I thought that was how to introduce `label_node_role_kubernetes_io` into the output.

I only have CodeReadyContainers deployment but the output looks like this:
```
{kernel_version="4.18.0-80.7.2.el8_0.x86_64",node="crc-56mmj-master-0",os_image="Red Hat Enterprise Linux CoreOS 410.8.20190807.0 (Ootpa)"}    1
```